pkg/audit/stats_reporter.go

```diff
@@ -0,0 +1,124 @@
+package audit
+
+import (
+	"context"
+	"time"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/metrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	totalViolationsName  = "violations_total"
+	constraintsTotalName = "constraints_total"
+	auditLatency         = "audit_latency"
+	methodType           = "audit"
```

Comment:
"dry_run" is also an option, right?

Reply:
added dry_run as an enforcement_action tag